third_party/move/scripts/move_pr.sh

```diff
@@ -143,31 +143,32 @@ if [ ! -z "$CHECK" ]; then
   )
 fi
 
+CARGO_OP_PARAMS="--locked --profile $MOVE_PR_PROFILE"
```

Comment:
That does not make sense, the parameter is definitely `--cargo-profile` to select the _cargo_ profile not the nexttest one. It is essential that we use the same profile for cargo build and nextest, which seems to be hampered by this PR. Maybe by accident there is still a nextest profile of the same name, but that is not what we want to select here.

See also here: https://nexte.st/docs/running/#options-and-arguments

Why should those change be necessary in the first place?

Reply:
As discussed off-line, this was needed due to an outdated version of `carg-nextest`.  I've updated this PR to add a comment to warn about this case and provide a solution, and also correctly read the `MOVE_PR_PROFILE` environment variable if it is defined.